docs/documentation/server_admin/topics/admin-console-permissions/fine-grain-v2.adoc

```diff
@@ -0,0 +1,362 @@
+themselves.
+
+*View and Manage users and group members*
+
+When evaluating *view* or *manage* permissions for users the group *view-members* and *manage-members* permissions are taken into an account.
+The evaluation follows:
+
+* Specific user/group permissions take precedence over broader all-resource permissions.
+* If multiple permissions apply to a given user or group (such as multiple user-specific permissions, or a permission covering a set of 
+  users/groups where the user is member of), all of them are evaluated, and all of them must grant access for the outcome to be GRANTED.
+* If no user/group-specific permissions exist, the evaluation falls back to all-resource permissions ("all-users", "all-groups").
+* When both all-users and all-groups permissions exist, both must grant access for the outcome to be GRANTED.
+* If only one of them exists, the outcome is determined by that permission alone.
+
+[[_realm_access_control]]
+==== Accessing a Realm Admin Console as a Realm Administrator
+
+Realm administrators can access a dedicated realm-specific Admin Console, which allows them to manage resources within their assigned realm. 
+This is separate from the main Keycloak Admin Console, which is typically used by server administrators.
+
+For more details on dedicated realm admin consoles and available roles, refer to: <<_per_realm_admin_permissions, Dedicated admin consoles>>.
+
+To access the Admin Console, a realm administrator must have at least one of the following roles assigned, depending on the resources they 
+need to administer:
+
+- *query-users* – Required to query realm users.  
+- *query-groups* – Required to query realm groups.  
+- *query-clients* – Required to query realm clients.  
+
+If an administrator is responsible for multiple resource types (such as both users and groups), they must have all corresponding "query-*" 
+roles assigned.
+
+These roles enable basic access to query resources but do not grant permission to view or modify them. To configure more fine-grained 
+administrative access, additional permissions must be granted using fine-grained admin permissions.
+
+===== Granting Administrative Roles to a Realm Administrator
+
+A realm administrator’s access must be configured by someone with permission to assign administrative roles. At a minimum, the administrator 
+must have:  
+
+- The appropriate "query-*" roles, depending on the resource types they need to administer.
+
+Beyond these foundational roles, *fine-grained admin permissions* can be used to define specific administrative capabilities. While fine-grained 
+permissions allow for more granular control over access, they cannot override the default behavior of built-in admin roles.
+This means that if an additional admin role is assigned to a realm administrator, permission evaluation will be bypassed, and access will be 
+granted.  
```

Comment:
Not sure I understand this.. if you grant any admin-role to a user, then permission evaluation is skipped irrespective of the role?

Reply:
The role should be respective to the action. I was trying to say, that if you assign for instance `manage-clients` role to your "realm-admin", no permissions related to client management are evaluated. 

So even if you have permission saying "do not allow to manage this client", the assigned role takes precedence. 

Would you suggest better wording to describe this behavior?

Reply:
For now, I have replaced "additional" by "respective".